### PR TITLE
Create pathogfair posts for Bluesky

### PR DESCRIPTION
**Pull request by arash77**

### CODE REVIEW

File: posts/2024/2024-07-16-pathogfair-preprint.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+---
+media:
+  - bluesky-galaxyproject
+
+hashtags:
+  bluesky-galaxyproject:
+    - metagenomics
+    - pathogen
+    - UseGalaxy
+
```
Comment (bebatut) [inline posts/2024/2024-07-16-pathogfair-preprint.md:10]: Are mentions possible?

**Review by bebatut — COMMENTED** [review]
Thanks for doing it

File: posts/2024/2024-07-16-pathogfair-preprint.md (hunk 1 above)
Comment (wm75) [reply posts/2024/2024-07-16-pathogfair-preprint.md:10]: I guess so, yes.

**Comment by wm75** [comment]
Ok then, I don't think we need to post this twice over there, but we know now that at least preview generation for bluesky works well and that outdated previews get collapsed when new ones get generated in a PR :-)

**Comment by bebatut** [comment]
> Ok then, I don't think we need to post this twice over there, but we know now that at least preview generation for bluesky works well and that outdated previews get collapsed when new ones get generated in a PR :-)

We could as it will be posted by Galaxy (more power than mine 😅 )

**Comment by wm75** [comment]
> We could as it will be posted by Galaxy (more power than mine 😅 )

Whatever power you can currently have on bluesky ;-)
Just make edits as needed then and tell us when it's good to go.

File: posts/2024/2024-07-16-pathogfair-preprint.md (hunk 1 above)
Comment (bebatut) [inline posts/2024/2024-07-16-pathogfair-preprint.md:10]:
```suggestion
mentions:
  bluesky-galaxyproject:
    - bebatut.bsky.social
```

**Comment by wm75** [comment]
is that how you mention people on bluesky? with their full id (even though there is only bsky.social?

**Comment by arash77** [comment]
> is that how you mention people on bluesky? with their full id (even though there is only bsky.social)?

Yes!

**Pull request closed without merging**